AppRestrictionSchema/Application/src/main/res/values/styles.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ Copyright 2014 The Android Open Source Project
```

Comment:
For new files, it should be
Copyright 2019 Google LLC
not AOSP version.

Reply:
Done